recipe/meta.yaml

```diff
@@ -8,26 +8,27 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/charset-normalizer-{{ version }}.tar.gz
-  sha256: f23667ebe1084be45f6ae0538e4a5a865206544097e4e8bbcacf42cd02a348f3
+  sha256: f30c3cb33b24454a82faecaf01b19c18562b1e89558fb6c56de4d9118a032fd5
 
 build:
   number: 0
-  noarch: python
```

Comment:
Considering that it's a dependency of conda, I'm worried that removing `noarch: python` will cause us problems. The rule for noarch is not a universal one. There are exceptions.

Reply:
Is the general rule if it's a dependency for conda we shouldn't change it to noarch?

Reply:
I've gone ahead and added back `noarch`.